Simplenote/src/main/res/layout/fragment_confirmation.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:gravity="center"
+    android:orientation="vertical">
+
+    <androidx.appcompat.widget.AppCompatImageView
+        android:layout_width="72dp"
+        android:layout_height="72dp"
+        android:layout_gravity="center"
+        android:layout_marginBottom="16dp"
+        android:src="@drawable/ic_mail"
+        android:tint="@color/gray_50" />
+
+    <TextView
+        android:id="@+id/email_confirmation_text"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="24dp"
+        android:layout_marginEnd="24dp"
+        android:layout_marginBottom="20dp"
+        android:gravity="center"
+        android:textColor="@color/gray_80"
+        android:textSize="16sp"
+        tools:text="We've sent an email to example@email.com. Please check your inbox and follow the instructions." />
+
+    <TextView
```

Comment:
It looks like there are a some custom components used throughout the simplenote app that are already styled for a consistent look. Maybe something to consider. Example: `com.automattic.simplenote.widgets.RobotoMediumTextView`

Reply:
This is now done here: cd8332561494a94bb3e13fe25059386075b91161